components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -125,6 +126,23 @@ func init() {
 	setConfigCmd.PersistentFlags().BoolVar(&configCmdFlags.postgresql, "pg", false, "Set toml configuration to the postgresql node[DUPLICATE]")
 	setConfigCmd.PersistentFlags().SetAnnotation("pg", docs.Compatibility, []string{docs.CompatiblewithHA})
 
+	//config oc-id-show-app flags
+	ocIdShowAppCmd.PersistentFlags().BoolVarP(&configCmdFlags.frontend, "frontend", "f", false, "Shows registered OCID app details from all frontend nodes")
```

Comment:
Yes, we need to check config in both automate and server node. In case the config is patched only in one place then there would be differences in the values that we get from automate and server nodes.

So we need the frontend flag as well.

Reply:
As per the discussion over call, we have removed the external flags.

The changes can be tracked in [Removed unnecessary flags from oc-id-show-app for automate HA #8060](https://github.com/chef/automate/pull/8060)

Resolving it.